<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>771</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>411</height>
     </rect>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="separator"/>
    <addaction name="actionSave_map"/>
    <addaction name="actionSave_Sheet"/>
   </widget>
   <widget class="QMenu" name="menuSpritesheet">
    <property name="title">
     <string>Spritesheet</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSpritesheet"/>
  </widget>
  <action name="actionSave_map">
   <property name="text">
    <string>Save map</string>
   </property>
  </action>
  <action name="actionSave_Sheet">
   <property name="text">
    <string>Save Sheet</string>
   </property>
  </action>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
